src/types/NodeType.js

```diff
@@ -0,0 +1,255 @@
+/* @flow */
```

Comment:
We could in principle wrap this up in a class, then make the consumer re-export. Would avoid the global singletons. May not matter.

Reply:
a Class? A factory function seems more obvious to me, I could be misunderstanding how you're thinking about here tho.

Reply:
either way. i mean, it'd be slightly nicer but it doesn't matter